wordy/wordy.spec.js

```diff
@@ -0,0 +1,89 @@
+import { WordProblem, ArgumentError } from './wordy';
+
+describe('Word Problem', () => {
+
+  it('add 1', () => {
+    const question = 'What is 1 plus 1?';
+    expect(2).toEqual(new WordProblem(question).answer());
```

Comment:
In the rest of tests so far, the expect line is like:

```
expect(new WordProblem(question).answer()).toEqual(2);
```

In fact, somebody opened a issue in xjavascript about swapped *expected* and *actual* value. This can be ported to xjavascript track, for this exercise.